test/scripts/jenkins_build_plugins.sh

```diff
@@ -4,10 +4,19 @@ source src/dev/ci_setup/setup_env.sh
 
 echo " -> building kibana platform plugins"
 node scripts/build_kibana_platform_plugins \
-  --oss \
   --scan-dir "$KIBANA_DIR/test/plugin_functional/plugins" \
   --scan-dir "$KIBANA_DIR/test/interpreter_functional/plugins" \
   --scan-dir "$KIBANA_DIR/test/common/fixtures/plugins" \
   --scan-dir "$KIBANA_DIR/examples" \
-  --workers 6 \
-  --verbose
+  --scan-dir "$KIBANA_DIR/test/plugin_functional/plugins" \
+  --scan-dir "$KIBANA_DIR/test/common/fixtures/plugins" \
+  --scan-dir "$XPACK_DIR/test/plugin_functional/plugins" \
+  --scan-dir "$XPACK_DIR/test/functional_with_es_ssl/fixtures/plugins" \
+  --scan-dir "$XPACK_DIR/test/alerting_api_integration/plugins" \
+  --scan-dir "$XPACK_DIR/test/plugin_api_integration/plugins" \
+  --scan-dir "$XPACK_DIR/test/plugin_api_perf/plugins" \
+  --scan-dir "$XPACK_DIR/test/licensing_plugin/plugins" \
+  --scan-dir "$XPACK_DIR/test/usage_collection/plugins" \
+  --scan-dir "$XPACK_DIR/test/security_functional/fixtures/common" \
+  --scan-dir "$XPACK_DIR/examples" \
+  --workers 12
```

Comment:
context on removing the `--verbose` flag?

Reply:
I can keep it as part of this PR, but I intend to remove all verbose output on CI. It's not helpful and it makes the logs too large and hard to find things that actually matter.